src/pages/Explorer/Explorer/explorer.module.scss

```diff
@@ -0,0 +1,242 @@
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    h3 {
+        font-family: 'Proxima Nova';
+        font-style: normal;
+        font-weight: 400;
+        font-size: 18px;
+        line-height: 26px;
+        color: var(--rhino);
+    }
+
+    .actions {
+        display: flex;
+        justify-content: center;
+
+        div:not(:last-child) {
+            margin-right: 8px;
+        }
+    }
+}
+
+.showMore {
+    width: 100%;
+    text-align: center;
+    padding: 13.5px 0;
+    border: 1px solid var(--porcelain);
+    border-top: none;
+    border-bottom-left-radius: 8px;
+    border-bottom-right-radius: 8px;
+    font-family: 'Proxima Nova';
+    font-style: normal;
+    font-weight: 400;
+    font-size: 14px;
+    line-height: 20px;
+    color: var(--jungle-green);
+    cursor: pointer;
+
+    &:hover {
+        background-color: var(--athens);
+    }
+}
+
+.row {
+    padding: 12px 24px;
+    width: 100%;
+    border: 1px solid var(--porcelain);
+    border-top: none;
+    cursor: pointer;
+
+    &:hover {
+        background-color: var(--athens);
+    }
+
+    .head {
+        margin-bottom: 10px;
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+
+        .title {
+            display: flex;
+            align-items: center;
+        }
+    }
+
+    .footer {
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+
+        .stat {
+            display: flex;
+            align-items: center;
+            justify-content: center;
+
+            .box {
+                display: flex;
+                align-items: center;
+                
+                &:not(:last-child) {
+                    margin-right: 15px;
+                }
+
+                .number {
+                    font-family: 'Proxima Nova';
+                    font-style: normal;
+                    font-weight: 400;
+                    font-size: 14px;
+                    line-height: 20px;
+                    color: var(--waterloo);
+                }
+            }
+        }
+
+        .user {
+            display: flex;
+            align-items: center;
+            cursor: pointer;
+
+            img {
+                width: 24px;
+                border: none;
+                border-radius: 50%;
+                margin-right: 8px;
+            }
+
+            .username {
+                font-family: 'Proxima Nova';
+                font-style: normal;
+                font-weight: 400;
+                font-size: 14px;
+                line-height: 20px;
+                color: var(--waterloo);
+            }
+
+            &:hover .username {
+                color: var(--jungle-green);
+            }
+        }
+    }
+
+    h2 {
+        font-family: 'Proxima Nova';
+        font-style: normal;
+        font-weight: 400;
+        font-size: 16px;
+        line-height: 24px;
+        margin-right: 8px;
```

Comment:
Use `san-webkit`'s typography class [`body-2`](https://github.com/santiment/san-webkit/blob/master/src/styles/text.css#L35) and margin class [`mrg-s mrg--r`](https://github.com/santiment/san-webkit/blob/master/src/styles/text.css#L35)